src/app/feedback/feedback.directive.ts

```diff
@@ -27,6 +27,7 @@ export class Feedback {
 })
 export class FeedbackDirective {
   message: string;
+  @Input() value: any;
```

Comment:
- `value` is a vague variable name that could confuse because it also has specific meaning on certain HTML elements.  Please call this `feedbackOf`
- To avoid having to check if this is undefined, you can give it a default value of `{}` by just assigning it: `@Input() feedbackOf: any = {}`